src/utils_project.js

```diff
@@ -157,7 +157,9 @@ go.utils_project = {
     is_valid_month: function(today, choiceYear, choiceMonth, monthsValid) {
         // function used to validate months for states 5A/5B & 12A/12B
 
-        var choiceDate = new moment(choiceYear+choiceMonth, "YYYYMM");
+        // NOTE: make sure we cast numbers to strings instead
+        //       of adding up numbers
+        var choiceDate = new moment('' + choiceYear + choiceMonth, "YYYYMM");
```

Comment:
Not sure if this is necessary but it seems too easy to insert integers like `2017` and `1` and have that get parsed as `2018` or something.